.flake8

```
@@ -1,6 +1,9 @@
 [flake8]
-ignore =
-    # W503 violates spec https://github.com/PyCQA/pycodestyle/issues/513
-    W503
-    # W504 has issues https://github.com/OCA/maintainer-quality-tools/issues/545
-    W504
+max-line-length = 88
```

Use
```suggestion
max-line-length = 79
```
plz per comments in the main thread.